index.html

```diff
@@ -0,0 +1,747 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+  <head>
+
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The New York Times Article</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.1/css/all.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+  </head>
+
+  <body class="Body-content">
+
+    <header class="page-top">
+
+      <nav class="top-nav">
+        <a class="nav-items" href="#"><img src="imgs/menu-bar.png" width="20" alt="menu icon"></a>
+        <a class="nav-items" href="#"><img src="imgs/search.png" width="20" alt="magnifying glass icon"></a>
+        <a class="nav-items" href="https://www.nytimes.com/section/science/space" id="space-link">SPACE & COSMOS</a>
+        <a class="nav-items" href="https://www.nytimes.com/" id="NYT-logo"><img src="imgs/NYT-Logo.png" width="260" alt="New York Times Logo"></a>
+        <button class="button" type="button">SUBSCRIBE NOW</button>
+        <button class="button" type="button">LOG IN</button>
+      </nav>
+
+      <section  class="intro">
+
+        <h1 class="article-title">Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+        <section class="social-media">
+          <h6>contact us</h6>
+          <nav>
+            <a href="#"><i class="fa fa-facebook"></i></a>
+            <a href="#"><i class="fa fa-whatsapp"></i></a>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+            <a href="#"><i class="fa fa-envelope"></i></a>
+            <a href="#"><i class="fa fa-share"></i></a>
+            <a href=""><img src="imgs/bookmark.png" width="20" alt="Bookmark Icon"></a>
```

Comment:
Let's add styles to the icon to look like the original page.
Original Page | Cloned Page
|-------------------|------------------|
![Screenshot from 2021-02-26 13-52-26](https://user-images.githubusercontent.com/44072711/109342588-14f57900-783a-11eb-9e16-e7b8528344bd.png) | ![Screenshot from 2021-02-26 13-51-01](https://user-images.githubusercontent.com/44072711/109342606-1aeb5a00-783a-11eb-9274-32238391ea87.png)